xarray/tests/test_dataarray.py

```diff
@@ -4778,3 +4778,14 @@ def test_weakref():
     a = DataArray(1)
     r = ref(a)
     assert r() is a
+
+
+def test_swap_dims_index_name():
```

Comment:
Can we modify `test_swap_dims` in this file instead? We should also modify `test_swap_dims` in  `test_dataset.py` too.

Can you also add a MultiIndex test case to `test_swap_dims` in both files.

Reply:
Done -- please let me know if there any other changes you think we need. Thanks for the quick review!